README.md

```diff
@@ -348,7 +348,16 @@ which means you can modify it, redistribute it or use it however you like.
 
 # CONFIGURATION
 
-You can configure youtube-dl by placing default arguments (such as `--extract-audio --no-mtime` to always extract the audio and not copy the mtime) into `/etc/youtube-dl.conf` and/or `~/.config/youtube-dl/config`. On Windows, the configuration file locations are `%APPDATA%\youtube-dl\config.txt` and `C:\Users\<user name>\youtube-dl.conf`.
+**On Mac:** You can configure youtube-dl by placing default arguments into `/etc/youtube-dl.conf` and/or `~/.config/youtube-dl.conf`.
```

Comment:
Umm, I think the correct term would be OSX. In any case, this is of course missing FreeBSD, OpenBSD, NetBSD, Solaris, OpenSolaris, and of course the primary platform, Linux. That's why it was written as "everyone uses ~, except Windows".